introduction/theTechnology.md

```diff
@@ -0,0 +1,59 @@
+<a href="https://www.surfly.com/">![logo](../images/logosmall.png)</a>
+
+# The Technology
```

Comment:
How about naming this something like "Comparison with other solutions"?
I love how they did it in Vue docs: https://vuejs.org/v2/guide/comparison.html
That is probably the most visited page in their docs